### DISCUSSION TITLE
Ubuntu harvester not connecting to Windows Farmer

### DISCUSSION THREAD

**Discussion by nomadengineer** [Chia-Network/chia-blockchain#3375] [Support]
Hi,
I have a full node setup on windows.
I am setting up an ubuntu 20.04 harvester.

Did all the relevant configurations on the ubuntu harvester:
- chia keys add
- copied and init with ca certs from main node
- updated config file with IP of main node as farmer_peer under harvester section.
- farmer_peer:
    host: 172.31.17.130
    port: 8447
- disabled upnp
- started harvester

I am not able to see any connection on the main node's windows chia gui, the log shows nothing.
The harvester log shows the following over and over.

2021-05-02T06:35:15.125 harvester harvester               : INFO     Reconnecting to peer {'host': '172.31.17.130', 'port': 8447}
2021-05-02T06:35:49.123 harvester harvester               : INFO     Reconnecting to peer {'host': '172.31.17.130', 'port': 8447}

I have checked all security rules. added exception in firewall.

**Comment by nomadengineer** [comment]
So it seems the main CHIA node needs to restart as well apparently for it to connect! 
I cannot add harvesters on the fly? :(

![image](https://user-images.githubusercontent.com/45670423/116805807-42b0b400-ab42-11eb-9c56-0d91c8f88681.png)

**Reply by danielduranc** [reply]
Hi, Could you solve it?, I want do it the same, but I don't have any idea.

**Reply by nomadengineer** [reply]
Hi,
So sometimes it does not show in the **Harvester Network** list, BUT its plots are showing up in the **Last Attempted Proof** section.
So if the number of plots in your harvester are visible in the **Last Attempted Proof** section as 0/xxx then you are fine!

**Reply by j88t** [reply]
hey man are you able to document how exactly did you do it?

**Reply by nomadengineer** [reply]
Follow this procedure.
https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines

**Comment by BrandtH22** [comment]
@nomadengineer , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
